src/backy/tests/test_main.py

```diff
@@ -105,22 +105,23 @@ def test_call_backup(tmpdir, capsys, caplog, argv, monkeypatch):
     with open(str(tmpdir / 'backy' / 'config'), 'wb') as f:
         f.write("""
 ---
-type: file
-filename: {}
+source:
+    type: file
+    filename: {}
 """.format(__file__).encode("utf-8"))
 
     monkeypatch.setattr(backy.backup.Backup, 'backup', print_args)
-    argv.extend(['-v', 'backup', 'test'])
+    argv.extend(['-v', 'backup', 'manual:test'])
```

Comment:
I just realize that we need to monitor for manual tags. We do have the global backy status check which ends up in Sensu, however, that's more about whether backy is creating new backups properly. Some way of reminding people that there are manual backups sitting around would be helpful. Let's extract that as a separate issue, though.